### PR TITLE
Feature new entry first

### LINKED ISSUE #1106: [Term Entry] Swift Arrays: `.first`
### Reason/inspiration (optional)

Please write a new entry under:
content/swiftui/concepts/views/terms/ about the Text view in SwiftUI.

A "Syntax" section with pseudo code snippet.
An "Example" section with a swift code snippet and an accompanying image to display the view.

To get started with the PR and entry file:

Under content/swift/concepts/arrays/terms/, create a new folder called first.
Then create a new file called first.md under content/swift/concepts/arrays/terms/first

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

Created new entry for .first() method and .first property in swift/concepts/arrays/terms.
Closes #1106 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### Additional information
I am using npm, I have node 18.12.
I was unable to run `format:verify` (prettier is not recognized)
I was unable to run `lint:md` (markdownlint is not recognized)
I was unable to run `test` (jest is not recognized)

### CODE REVIEW

File: content/swift/concepts/arrays/terms/first/first.md
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/first/first.md:46]:
```suggestion
## The `.first` Property
```

File: content/swift/concepts/arrays/terms/first/first.md
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/first/first.md:66]:
```suggestion
In the example above, the `.first` property of the `numbers` array is printed out. The `.first` property will return the first item of the `numbers` array as an optional value. The `!` forces the expression to unwrap the value of the returned first element. This will output:
```

File: content/swift/concepts/arrays/terms/first/first.md
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/first/first.md:50]:
```suggestion
### Syntax
```

File: content/swift/concepts/arrays/terms/first/first.md
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/first/first.md:58]:
```suggestion
### Example
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla, I did a first review and left a couple of comments.

**Comment by cslylla** [comment]
Thanks, @SSwiniarski, I reviewed and accepted the suggested changes.

**Review by SSwiniarski — APPROVED** [review]
@cslylla, This is going to go through a second review, then once that review is done and approved, one of the maintainers will merge this PR.

File: content/swift/concepts/arrays/terms/first/first.md
Comment (yangc95) [inline content/swift/concepts/arrays/terms/first/first.md:66]: Force unwrapping isn't good practice. Maybe we can include a note about this!

**Review by yangc95 — COMMENTED** [review]
@cslylla went ahead and did a second review! Please tag me once you've looked over my comments/suggestions.

File: content/swift/concepts/arrays/terms/first/first.md
Comment (cslylla) [reply content/swift/concepts/arrays/terms/first/first.md:67]: Sure, I will add a note about that.

File: content/swift/concepts/arrays/terms/first/first.md
Comment (cslylla) [reply content/swift/concepts/arrays/terms/first/first.md:29]: Yes, that is a good idea.

File: content/swift/concepts/arrays/terms/first/first.md
Comment (cslylla) [reply content/swift/concepts/arrays/terms/first/first.md:27]: I will add it to the pseudo-code.

**Review by cslylla — COMMENTED** [review]
I tried to implement the changes you requested, @yangc95.

**Review by yangc95 — APPROVED** [review]

**Comment by Dusch4593** [comment]
@cslylla Congrats on making your first Docs contribution! 🎊

I was wondering if you were open to making another contribution to this entry with a new PR? I wanted to suggest the following changes:

- On line # 15, I think this opening sentence can be consolidated a bit more into the following:
  > The \*\*\`.first()\`\*\* method returns the first element of the array that fulfills the condition. Otherwise, it returns `nil`.
- Under the "Syntax" section, lines # 25-27, can we combine this text into 1-2 sentences (no list item)? Perhaps something like the following:
  > The \`.first()\` method takes a single \`condition\` parameter. It is a [closure](https://www.codecademy.com/resources/docs/swift/closures) that returns a Boolean value indicating whether a given array element is a match.
  - Under the "Syntax" section, _parameter_ is usually preferred because it represents value(s) would be expected to be passed as arguments when the function/method is called.
 - In the very next section (the first "Example"), inside the \```swift code block, let's add spacing between the array elements on line # 32 so it matches the spacing of the array on line # 62.
 - On line # 39, I would suggest rephrasing the sentence into the following:
   >  In the example above, the `.first()` method is searching through the `values` array. The first item that meets the condition of being greater than `5` will be returned.

**Comment by cslylla** [comment]
Hey @Dusch4593, thanks for your feedback. I created a new PR (#1308 ) with the changes you suggested.

**Comment by Dusch4593** [comment]
@cslylla Thank you! Will take a look soon. 😄

**Pull request merged**
